packages/cardano-services/src/StakePool/DbSyncStakePoolProvider/types.ts

```diff
@@ -124,8 +124,8 @@ export interface SubQuery {
   query: string;
 }
 
-export interface TotalAdaModel {
-  total_ada: string;
+export interface ReservesModel {
```

Comment:
_Reserves_ is a specific pot of funds in the network. This is the total UTxOs value right?

Reply:
Ah, the diff is confusing me :-)